package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@inscreen/html-to-image",
-  "version": "1.11.12",
+  "version": "1.12.0",
```

Comment:
Did they also release `1.12.0`? Let's not give random numbers to versions that conflict with _their_ versions, if we continue to incorporate changes from them. I'm pretty sure npm allows specifying more than three numbers in the version (e.g. `-is2`, `-is3`, or something else).